.github/ISSUE_TEMPLATE/proposed-feature.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Proposed Feature
+about: Propose a solution to a problem or wishlist item
+title: ''
+labels: proposed-feature
+assignees: ''
```

Comment:
Not sure what the difference between this and a feature request is.

Reply:
I agree with @mkindahl ... can we simplify it and have just bug report and feature request template??

Reply:
agree; I've copied the templates from `timescaledb-toolkit` and kept which seemed relevant.

I've removed this template